api/index.ts

```diff
@@ -0,0 +1,3 @@
+import app from "../src";
+
+export default app;
```

Comment:
I'm just curious what's the reason for this file if it exports the same server from the index.ts from src

Reply:
The way that vercel's serverless functions work is that they map anything instead the api directory to a serverless func. So this was a hacky way of importing the app we had defined with express, and then just exporting it again so that it shows up as a candidate to be converted into a serverless function.